src/NServiceBus.AcceptanceTests/Routing/When_replying_to_message_with_interface.cs

```diff
@@ -0,0 +1,112 @@
+﻿namespace NServiceBus.AcceptanceTests.Routing
+{
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using AcceptanceTesting.Customization;
+    using EndpointTemplates;
+    using NUnit.Framework;
+
+    public class When_replying_to_message_with_interface_and_unobtrusive : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_reply_to_originator()
+        {
+            var ctx = await Scenario.Define<Context>()
+                .WithEndpoint<SendingEndpoint>(c => c
+                    .When(b => b.Send(new MyMessage())))
+                .WithEndpoint<ReplyingEndpoint>()
+                .WithEndpoint<OtherEndpoint>()
+                .Done(c => c.SendingEndpointGotResponse)
+                .Run();
+
+            Assert.IsTrue(ctx.SendingEndpointGotResponse);
+            Assert.IsFalse(ctx.OtherEndpointGotResponse);
+        }
+
+        [Test]
+        public async Task Should_reply_to_configured_return_address()
```

Comment:
Should we remove this one since it is not really related to mapping and unobtrusive?